src/tray/tray.yml

```
@@ -0,0 +1,3 @@
+metadata:
+  name: joanie
+  version: main
```

If you plan the first release for a major one:

```suggestion
  version: 1.0.0
```